src/geodesic.c

```diff
@@ -85,7 +85,10 @@ static void Init() {
     tolb = tol0 * tol2;
     xthresh = 1000 * tol2;
     degree = pi/180;
-    NaN = sqrt(-1.0);
+    {
```

Comment:
@cffk See above. One of the versions of cppcheck complains that sqrt(-1.0) is undefined, but later no longer.